src/main/java/com/ericsson/ei/utils/TextFormatter.java

```diff
@@ -0,0 +1,25 @@
+package com.ericsson.ei.utils;
+
+
+/**
+ * Class that handles and formatting Strings for different needs
+ * and can be used in different classes.
+ *
+ */
+public class TextFormatter {
+    
+    /**
+     * Function that removes ENC parentheses from encrypted string.
+     * Commonly used for password properties that has a format "ENC(d23d2ferwf4t55)"
+     * This function removes "ENC(" text and end ")" parentheses and return only the
+     * encryption as string.
+     * 
+     * @param stringWithEncryptionParantheses  The string that contains the ENC() string.
+     */
+    public String removeEncryptionParentheses(String stringWithEncryptionParantheses) {
+        String formattedEncryptionString = stringWithEncryptionParantheses.replace("ENC(","");
+        int lastParanthesesOccurenxeIndex = formattedEncryptionString.lastIndexOf(")");
```

Comment:
If no occurence of ")" exists, this will return -1. In subsequence below, will that give some, out of index error?

Reply:
Changed so if user forgot end parentheses in configuration, "ENC(dsfgds3244r3" , it will return the password string without "ENC(" and skip trying replace last end parentheses.
Pushing code change + tests in next commit.